src/NumberTheory/QuadFormAndIsom/enumeration.jl

```diff
@@ -281,21 +350,27 @@ function is_admissible_triple(
   for s in _Cp
     s[1] += 2
   end
-  Cp = ZZLocalGenus(p, _Cp)
+  Cp1 = ZZLocalGenus(p, _Cp)
 
   # Condition (5) of Definition 4.13
-  if !represents(local_symbol(AperpB, p), Cp)
+  if !represents(local_symbol(AperpB, p), Cp1)
     return false
   elseif !represents(C, AperpB)
     return false
   end
 
-  # Condition (4) of Definition 4.13
-  qC = discriminant_group(C)
-  special = (p == 2) && (_is_free(qA, p, l+1)) && (_is_free(qB, p, l+1)) && (_is_even(qC, p, l))
-  rA = _rho_functor(qA, p, l+1; quad=special)
-  rB = _rho_functor(qB, p, l+1; quad=special)
-  return is_anti_isometric_with_anti_isometry(rA, rB)[1]
+  # Condition (4) of Definition 4.13 
+  _is_anti_isometric_bilinear(Ap, Bp, l+1) || return false 
+    
```

Comment:
```suggestion
    p != 2 && return true
```
The rest is only for `p` equal to 2